cpp/tests/streams/replace_test.cpp

```diff
@@ -0,0 +1,109 @@
+}
+
+TEST_F(ReplaceTest, ReplaceNullsPolicy)
+{
+  cudf::test::fixed_width_column_wrapper<int> input({{0, 0, 0, 0, 0}, {0, 0, 1, 1, 1}});
+  cudf::replace_nulls(input, cudf::replace_policy::FOLLOWING, cudf::test::get_default_stream());
+}
+
+TEST_F(ReplaceTest, ReplaceNansColumn)
+{
+  auto nan          = std::numeric_limits<double>::quiet_NaN();
+  auto input_column = cudf::test::make_type_param_vector<double>({0.0, 0.0, nan, nan, nan});
+  cudf::test::fixed_width_column_wrapper<double> input(input_column.begin(), input_column.end());
+  cudf::test::fixed_width_column_wrapper<double> replacement({0, 1, 2, 3, 4});
+  cudf::replace_nans(input, replacement, cudf::test::get_default_stream());
+}
+
+TEST_F(ReplaceTest, ReplaceNansScalar)
+{
+  auto nan          = std::numeric_limits<double>::quiet_NaN();
+  auto input_column = cudf::test::make_type_param_vector<double>({0.0, 0.0, nan, nan, nan});
+  cudf::test::fixed_width_column_wrapper<double> input(input_column.begin(), input_column.end());
+  auto replacement = cudf::numeric_scalar<double>(4, true, cudf::test::get_default_stream());
+  cudf::replace_nans(input, replacement, cudf::test::get_default_stream());
+}
+
+TEST_F(ReplaceTest, FindAndReplaceAll)
+{
+  cudf::test::fixed_width_column_wrapper<int> input({0, 0, 0, 0, 0});
+  cudf::test::fixed_width_column_wrapper<int> values_to_replace({0, 0, 0, 0, 0});
+  cudf::test::fixed_width_column_wrapper<int> replacement_values({1, 1, 1, 1, 1});
+  cudf::find_and_replace_all(
+    input, values_to_replace, replacement_values, cudf::test::get_default_stream());
+}
+
+TEST_F(ReplaceTest, ClampWithReplace)
+{
+  cudf::test::fixed_width_column_wrapper<int> input({0, 1, 2, 3, 4, 5, 6, 7, 8, 9});
+  auto low          = cudf::numeric_scalar<int>(3, true, cudf::test::get_default_stream());
+  auto low_replace  = cudf::numeric_scalar<int>(5, true, cudf::test::get_default_stream());
+  auto high         = cudf::numeric_scalar<int>(7, true, cudf::test::get_default_stream());
+  auto high_replace = cudf::numeric_scalar<int>(6, true, cudf::test::get_default_stream());
+  cudf::clamp(input, low, low_replace, high, high_replace, cudf::test::get_default_stream());
+}
+
+TEST_F(ReplaceTest, Clamp)
+{
+  cudf::test::fixed_width_column_wrapper<int> input({0, 1, 2, 3, 4, 5, 6, 7, 8, 9});
+  auto low  = cudf::numeric_scalar<int>(3, true, cudf::test::get_default_stream());
+  auto high = cudf::numeric_scalar<int>(7, true, cudf::test::get_default_stream());
+  cudf::clamp(input, low, high, cudf::test::get_default_stream());
+}
+
+TEST_F(ReplaceTest, NormalizeNansAndZeros)
+{
+  auto nan          = std::numeric_limits<double>::quiet_NaN();
+  auto input_column = cudf::test::make_type_param_vector<double>({-0.0, 0.0, -nan, nan, nan});
+  cudf::test::fixed_width_column_wrapper<double> input(input_column.begin(), input_column.end());
+  cudf::normalize_nans_and_zeros(static_cast<cudf::column_view>(input),
```

Comment:
```suggestion
  cudf::normalize_nans_and_zeros(input,
```
Just a minor nit. I don't think this cast is needed.